daml-lf/tests/scenario/stable/mustfails/EXPECTED.ledger

```diff
@@ -85,4 +85,4 @@ TX #21 1969-12-31T23:58:20Z [Test:345] version: 12
 mustFailAt actAs: {'Alice'} readAs: {} [Test:346]
 
 active contracts:
-   00049c3d61768cf931aa2b97a7ac28002d7673c3f8b996798d87227106e178286e, 00067f889bdf49aa670596cab1a5907d75596d971df3477087c88ca86f281cee85, 00154fe9827fdb40afae4ba484f3f51159d99a27c1463928ec761043c28353b76c, 007e5c9342585588fff0855b297c6e06bfb6a3d62578a28da860af02a98048bd68, 00945c0656f04cdd9f6a85cb466acb0edcd75ea0f618362e83b92164c4cee523e9, 00bf46b75494167be56ef270a63cbc740370b75c98fb517457d9859d55796f7fbf, 00e52b332d66bbf5ac835144e35bf339a7ab319e505e005d1ddd9aed313764dd51
+   00067f889bdf49aa670596cab1a5907d75596d971df3477087c88ca86f281cee85, 005c2e250ff7988dbda72348b8172716801c6ad40cacecba387863315a8ed678db, 009097ab3d05b0538d961df2cb88558e4cd1947514d2f2c275363a3dc81e07b6aa, 00945c0656f04cdd9f6a85cb466acb0edcd75ea0f618362e83b92164c4cee523e9, 00a5627208a832b7d951562d793cb1c9e48b3459603d2543b48656ea3ac2debf41, 00d34fd9f2c2e66e6d68c344f1bec486f1c5624a4268bae020e2584a1ad411d65d, 00e52b332d66bbf5ac835144e35bf339a7ab319e505e005d1ddd9aed313764dd51
```

Comment:
I spent quite some time convincing me that this is correct so let me record this:

The old code was broken. We hit https://github.com/digital-asset/daml/blob/6c919b3877dcc46e97477e8e75107bd3ef85cb4c/daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala#L445 for cases like empty controllers. We then call `abortExercises` in a non-exercise context. This overwrites the `aborted` field and we end up with some garbage. This PR fixes this by not adding the close until we have the open.